core/fortran-example/tutorial/05_parallel_for/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+add_executable(ForParallelFor Fortran_main.f90 Fortran_kernels.F90 parallel_for_example.cpp)
+target_link_libraries(ForParallelFor cabanacore)
+add_test(NAME Fortran_tutorial_05 COMMAND ForParallelFor)
+set_property(TARGET ForParallelFor PROPERTY LINKER_LANGUAGE Fortran)
+
+message(${CMAKE_Fortran_COMPILER_ID})
+if (("${CMAKE_Fortran_COMPILER_ID}" STREQUAL "PGI") AND ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "PGI"))
+  if( Cabana_ENABLE_Cuda )
+    target_compile_definitions(ForParallelFor PUBLIC -DUSE_GPU=1)
+    set(CMAKE_Fortran_FLAGS "-ta=tesla:cuda9.0 -Minfo=accel -Mcuda=cuda9.0")
+    set(CMAKE_CXX_FLAGS "--relocatable-device-code=true")
+  elseif( Cabana_ENABLE_OpenMP )
+    target_compile_definitions(ForParallelFor PUBLIC -DUSE_GPU=0)
+    set(CMAKE_Fortran_FLAGS "-ta=multicore -mp")
+  elseif( Cabana_ENABLE_Serial )
+    target_compile_definitions(ForParallelFor PUBLIC -DUSE_GPU=0)
+    set(CMAKE_Fortran_FLAGS "-ta=host")
+  endif()
+  
+  set(CMAKE_EXE_LINKER_FLAGS "-pgc++libs")
```

```suggestion
  if(CMAKE_VERSION VERSION_LESS_EQUAL 3.13)
    set(CMAKE_EXE_LINKER_FLAGS "-pgc++libs")
  else()
    target_link_options(ForParallelFor "-pgc++libs")
  endif()
```
from the the teleconf today.

---

done.